common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/TokenRequest.java

```diff
@@ -201,9 +236,13 @@ public void setRefreshToken(final String refreshToken) {
     }
 
     public static class GrantTypes {
-        public final static String AUTHORIZATION_CODE = "authorization_code";
-        public final static String REFRESH_TOKEN = "refresh_token";
-        public final static String PASSWORD = "password";
+        public static final String AUTHORIZATION_CODE = "authorization_code";
+        public static final String REFRESH_TOKEN = "refresh_token";
+        public static final String PASSWORD = "password";
+    }
+
+    public static class TokenType {
+        public static final String POP = "pop";
```

Comment:
Do we need a constant for Bearer?

Reply:
Not for any explicit reason, no. We have to specify a `token_type == pop` for ESTS to service the `/token` request with a `req_cnf` claim. This param doesn't exist (or is assumed) for the default bearer flow so it's not necessary to specify when it's `Bearer`:
https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-access-token